src/constants/minions.js

```diff
@@ -274,6 +274,7 @@ export const minions = {
     name: "Voidling",
     type: "combat",
     head: "/head/3a851ed2ce5c2c0523af772d206d9555e2e1383ec87946e6ff4c51186e29ef7f",
+    tiers: 12,
```

Comment:
Voidling has 11 tiers, in game crafted generators is bugged

Reply:
@dukio is this resolved?

Reply:
@nstringham yes